modules/nf-core/sam2lca/analyze/main.nf

```diff
@@ -0,0 +1,43 @@
+
+process SAM2LCA_ANALYZE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::sam2lca=1.1.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sam2lca:1.1.2--pyhdfd78af_1':
+        'quay.io/biocontainers/sam2lca:1.1.2--pyhdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
```

Comment:
Maybe change this to alignment and index?

Reply:
I've used bam and bai because it seems most modules use the same terminology :)

Reply:
Does it support cram?